deploy/charts/harvester/dependency_charts/snapshot-validation-webhook/templates/deployment.yaml

```diff
@@ -27,7 +27,8 @@ spec:
             {{- end }}
           ports:
             - name: https
-              containerPort: {{ .Values.service.port | default 443 }}
+              containerPort: {{ .Values.args.port | default 8443 }}
+              protocol: TCP

```

Comment:
You mentioned:

> I follow the example like https://github.com/piraeusdatastore/helm-charts/tree/snapshot-validation-webhook-1.9.0/charts/snapshot-validation-webhook to change container port or add --port=8443 to the argument, but it can't fix the error.

Given Gaurav's subsequent fix, is this port change necessary?  If not, I'd suggest to revert it (unless there's some other reason I'm not yet aware of to change the port)

Reply:
Yeah, I think this update is not helpful. Reverted it. Thanks.